app/core/p2p/sync_chain.py

```diff
@@ -572,6 +572,10 @@ def find_forking_block(node_url):
 
 def quick_sync(db_url):
     """Copies the entire newrl.db from a peer. Unsecured."""
+    logger.info("Syncing using quick-sync")
+    if NETWORK_SNAPSHOT:
```

Comment:
@Prannav-k why use this if statement?

Reply:
if its any network where trusted central snapshot isnt configured, it will take from already selected random node.